src/devices/ExplorerHat/ExplorerHat.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.ExplorerHat
+{
+    /// <summary>
+    /// Pimoroni Explorer HAT for Raspberry Pi
+    /// </summary>
+    public class ExplorerHat : IDisposable
+    {
+        private readonly bool _shouldDispose;
+        private GpioController _controller;
+
+        /// <summary>
+        /// Explorer HAT DCMotors collection
+        /// </summary>
+        public Motors Motors { get; private set; }
+
+        /// <summary>
+        /// Explorer HAT led array
+        /// </summary>
+        public Lights Lights { get; private set; }
+
+        /// <summary>
+        /// Initialize <see cref="ExplorerHat"/> instance
+        /// </summary>
+        public ExplorerHat(GpioController controller = null, bool shouldDispose = true)
+        {
+            _controller = controller;
+            _shouldDispose = shouldDispose;
+
+            if (_controller is null)
+            {
+                _controller = new GpioController(PinNumberingScheme.Logical);
+            }
+
+            if (_controller.NumberingScheme != PinNumberingScheme.Logical)
```

Comment:
where is this coming from? We don't do similar checks elsewhere, except we provide way to override pins. I'd just remove the check and potentially in the future if needed we can provide ways to override pins

Reply:
I did it because receiving a controller initialized with NumberingScheme.Board, things doesn't works, but I like the "provide ways to override pins" concept.